distribution/docker/build.gradle

```diff
@@ -236,6 +239,8 @@ subprojects { Project subProject ->
 
     exportDockerImageTask.dependsOn(parent.tasks.getByName(buildTaskName))
 
+    exportDockerImageTask.onlyIf { Architecture.current().name().toLowerCase().equals(architecture) }
```

Comment:
Hmmm, don't we already have an issue where this blows up if we don't have Docker installed?

Reply:
I don't know. The reason I added this was to stop the build from potentially exporting a Docker image of the wrong arch, from a previous build. I do think we should put the arch in the tag name, and we'll likely have to in order to create multi-arch builds.